src/scss/_standard.scss

```diff
@@ -1,27 +1,161 @@
+////
+/// @group o-card base styles
+/// @link http://registry.origami.ft.com/components/o-card
+////
+
+/// Base card styles
 @mixin oCard {
 	@include oColorsFor(o-card box, background);
-	height: 100%;
-	position: relative;
+	width: 100%;
+	padding: $o-card-spacing-unit;
+	padding-bottom: 0;
+	display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+}
+
+/// Base tag styles
+@mixin oCardTag {
+	@include oColorsFor(o-card-tag, text);
+	@include oTypographySansBold(s);
```

Comment:
Have you had time to see if there's a way to disable custom font loading when using o-typography?